autoload/neomake/makers/ft/typescript.vim

```diff
@@ -7,8 +7,9 @@ endfunction
 function! neomake#makers#ft#typescript#tsc()
     return {
         \ 'args': [
-            \ '-m', 'commonjs', '--noEmit'
+            \ '--project', getcwd(), '--noEmit'
```

Comment:
The `getcwd()` looks weird..  it should use the buffer's dir or something probably?
`getcwd()` might return e.g. a local cwd.

Do we need something to get a project's root for this?  (see e.g. https://github.com/dbakker/vim-projectroot)